### ISSUE TITLE
filing financials: some facts are missing when converting income statement to dataframe

### ISSUE THREAD

**Issue by ilias-ant** [dgunning/edgartools#49]
Python version: `3.12.2 (main, Mar 12 2024, 08:01:18) [GCC 12.2.0]`
edgartools version: `2.21.0`

**Minimum Reproducible Example**

```python
from edgar import Company, set_identity


if __name__ == "__main__":
    set_identity("Ilias Antonopoulos (xxxxxxxxxx@gmail.com)")

    filings = Company("AAPL").get_filings(form="10-K", filing_date="2023-01-01:2023-12-31")

    filing = filings.get(0).obj()

    print(filing.financials.income_statement)
    print(filing.financials.income_statement.to_dataframe())

```

I have noticed that, although the facts present in `filing.financials.income_statement` are correct, when compared with the corresponding [income statement table from SEC](https://www.sec.gov/Archives/edgar/data/320193/000032019323000106/aapl-20230930.htm), the facts:

- Total Net Sales
- Cost Goods and Services Sold
- Selling General and Administrative Expenses (*a bit more minor*)

are missing from the `.to_dataframe()` version of it (*see screenshot below*).

<img width="710" alt="image" src="https://github.com/dgunning/edgartools/assets/42189572/853db7e7-91e2-4949-9c32-35eb57533b12">

**Comment by ilias-ant** [comment]
Also, noticed a similar issue regarding the `Common Stock` fact of Nvidia's 2023 balance sheet:

```python
from edgar import Company, set_identity

if __name__ == "__main__":
    set_identity("Ilias Antonopoulos (xxxxxxxxx@gmail.com)")

    filings = Company("NVDA").get_filings(form="10-K", filing_date="2023-01-01:2023-12-31")

    filing = filings.get(0).obj()

    print(filing.financials.balance_sheet)
    print(filing.financials.balance_sheet.to_dataframe())
```

<img width="733" alt="Screenshot 2024-05-05 at 11 38 11 PM" src="https://github.com/dgunning/edgartools/assets/42189572/52fbfc85-5f9e-4f64-9b0e-a55d4a88897a">

**Comment by dgunning** [comment]
Fixed in 2.21.1
- missing items in dataframe
- mostly empty columns included in financial tables
